tonytanaka/index.js

```diff
@@ -0,0 +1,16 @@
+// Question 1
+// NaN stands for Not-A-Number
+let ex1a = "hello" * 1;
+console.log(typeof ex1a);
+let ex1b = "100em";
+console.log(isNaN(ex1b));
```

Comment:
Also here in line 6 you have to check if ex1b is NaN or not! And result should be true or false!
console.log(isNaN(ex1b));

Reply:
this line already exists

Reply:
Oh indeed, my bad!